service/src/main/java/uk/gov/hmcts/reform/fpl/utils/DocumentUploadHelper.java

```diff
@@ -16,10 +16,19 @@
     private final RequestData requestData;
 
     public String getUploadedDocumentUserDetails() {
-        UserDetails userDetails = idamClient.getUserDetails(requestData.authorisation());
+        UserDetails userDetails = getUserDetails();
 
         boolean isHmctsUser = userDetails.getRoles().stream().anyMatch(UserRole::isHmctsUser);
 
         return isHmctsUser ? "HMCTS" : userDetails.getEmail();
```

Comment:
As you can see this will always return HMCTS for you which I don't think is wanted, the user service will provide you with what you want (and is a much more sensible class to use)